### PR TITLE
Update bleve dependency to latest master revision

### PR DESCRIPTION
**Pull request by lunny**
will unblock #5044

### CODE REVIEW

File: Gopkg.toml [hunk 1]
```diff
@@ -15,10 +15,8 @@ ignored = ["google.golang.org/appengine*"]
   name = "code.gitea.io/sdk"
 
 [[constraint]]
-#  branch = "master"
-  revision = "c74e08f039e56cef576e4336382b2a2d12d9e026"
+  branch = "master"
   name = "github.com/blevesearch/bleve"
-#Not targetting v0.7.0 since standard where use only just after this tag
```
Comment (techknowlogick) [inline Gopkg.toml:21]: Near bottom of this file you can remove constraint on github.com/go-gitea/bolt"

Comment (lunny) [reply Gopkg.toml:21]: done.

Comment (lafriks) [reply Gopkg.toml:21]: I think we should still fix revision to have predictable build

Comment (lunny) [reply Gopkg.toml:21]: @lafriks done.

**Review by lafriks — APPROVED** [review]

**Review by adelowo — APPROVED** [review]

**Comment by zeripath** [comment]
Needs a milestone setting... 1.8.0 or 1.9.0?

**Comment by techknowlogick** [comment]
I’m going to set as 1.8 because it is a requirement to make this switch for go modules

**Pull request merged**